leaders/README.md

```diff
@@ -1 +1,69 @@
-# Notes for course leaders
+# Checklist for course leaders
+
+If this looks like too much: **delegate, delegate, delegate**. 
+
+Just because you are leading the week, it does not mean that you have to do all the work for it. A lot of the interactions during the week are short, do not need preparation, and fun.  
+
+Do not hesitate to plunder material from previous courses and do not take on more work that you can comfortably handle. Work in pairs or threes or even groups of four, but make sure that at least one person each day is responsible for keeping to the timetable.
+
+## In advance
++ **Actions to take**:
+  + Meet with the previous week's course leaders
+  + Check [Stop, Go, Continue notes](../sgc/README.md) for any change requests
+  + Tweak the [timetable](../timetable) for your week
+  + Check if any [learning outcomes from previous week](../patterns) still need completing
+  + Update the relevant pages on the book to reflect the new content
++ **Materials to prepare and things to organise**:
+  + Weekend assignment for the previous Friday (what do students need to research for your week?)
+  + All-day workshop for Monday
+  + Topics for Tuesday (no more than 16 learning outcomes total)
+  + Speakers and MVP pitches for Monday and Tuesday
+  + Morning challenges for Tuesday and Wednesday
+  + Project for Wednesday
+  + Find volunteers for office hours on Wednesday and Thursday
+  + Find volunteer for Friday upstairs project presentation
+  + External reviewer for Friday code review
+
+## End of the week
++ Update [Stop, Go, Continue notes](../sgc/README.md)    
++ Review this checklist    
++ Review the course content and amend as necessary   
++ Hand-over to next week's course leaders
+  
+## Contact time
+
+### Monday
+Workshop, as required    
+17:00 Introduce invited speaker    
+
+### Tuesday
+10:00 Introduce morning challenge    
+10:45 Review morning challenge    
+13:00 Research presentations #1    
+16:00 Research presentations #2    
+17:00 Introduce invited speaker    
+
+### Wednesday
+10:00 Introduce morning challenge    
+10:45 Review morning challenge    
+11:00 Project kick-off    
+17:00 Office hours    
+18:00 Run learning points review    
+
+### Thursday
+13:00 Run mid-point project reviews     
+17:00 Office hours    
+
+### Friday
+10:00 Code review kick-off    
+11:30 Stop code reviews give people a chance to respond to issues    
+13:00 Announce Stop, Go, Continue    
+13:30 Stop, Go, Continue review    
+14:15 Introduce Upstairs project presentation    
+14:45 Introduce weekend assignments    
```

Comment:
@iteles I think yes, but it should be kept light, i.e. mostly tidying up issues and doing a bit of reading and watching.